src/clait/format/templates/french_autoentrepreneur.py

```diff
@@ -177,7 +177,8 @@
 \ifthenelse{{\equal{{\FactureAcquittee}}{{paid}}}}{{
     Facture acquittée.
 }}{{
-    Facture à régler, par chèque ou par virement bancaire :
+    Paiement à 60 jours suivant réception de la facture par chèque ou virement
```

Comment:
My friend mentioned me this link:


https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000005634379&idArticle=LEGIARTI000019294314

> Le délai convenu entre les parties pour régler les sommes dues ne peut dépasser soixante jours à compter de la date d'émission de la facture. Par dérogation, un délai maximal de quarante-cinq jours fin de mois à compter de la date d'émission de la facture peut être convenu entre les parties, sous réserve que ce délai soit expressément stipulé par contrat et qu'il ne constitue pas un abus manifeste à l'égard du créancier. En cas de facture périodique, au sens du 3 du I de l'article 289 du code général des impôts, le délai convenu entre les parties ne peut dépasser quarante-cinq jours à compter de la date d'émission de la facture.